_includes/specials/taproot/en/20-activation.md

```diff
@@ -0,0 +1,66 @@
+Less than two weeks after the publication of this post, taproot is
+expected to activate at block {{site.trb}}.  We know what we expect to
+happen, and we also know about a few possible failure modes.
+
+The best and most likely outcome is that everything goes fine.  Nothing
+that happens should be visible to normal users.  Only those carefully
+monitoring their nodes or trying to create taproot transactions should
+notice anything.  At block 709,631 nearly all full nodes we're aware of
+will be enforcing the same consensus rules.  One block later, nodes
+running Bitcoin Core 0.21.1, 22.0, or related releases will be enforcing
+the additional [taproot][topic taproot] rules not enforced by earlier
+software releases.
+
+This carries a risk that earlier and later versions of node software
+will accept different blocks.  This happened during the activation of
+the [BIP66][] soft fork in 2015, resulting in a six-block chainsplit and
+several shorter chainsplits.  Significant engineering effort has been
+invested in preventing a repeat of that problem, and a similar problem
+should only happen with taproot if a miner either deliberately mines a
+taproot-invalid block or has disabled safety measures hardcoded into
+Bitcoin Core and related node software.
+
+Specifically, to create a chainsplit, a miner would need to create or
+accept a transaction that spends from a taproot output (segwit v1
+output) without following taproot's rules.  If a miner did that, they
+would lose at least 6.25 BTC (about $400,000 USD at time of writing) if
+the economic consensus of Bitcoin node operators rejects the
+taproot-invalid blocks.
+
+We can't know for sure without creating an invalid block what those node
+operators will do---nodes can be run entirely privately---but [proxy
+measures][bitnodes] indicate that perhaps more than 50% of node
+operators are running taproot-enforcing versions of Bitcoin Core.
+That's probably more than sufficient to ensure any miner who creates a
+taproot-invalid block will be fully penalized.
```

Comment:
"fully penalized" doesn't feel right to me, since it implies both that a miner may be partially penalized, and that there's someone actively doing the penalizing.

Perhaps something like:

```suggestion
taproot-invalid block will see their block rejected by the network and will not be able to collect their reward for the work they've done.
```